src/components/containers/Fold.js

```diff
@@ -84,7 +84,7 @@ Fold.propTypes = {
   name: PropTypes.string,
   type: PropTypes.string,
   className: PropTypes.string,
-  icon: PropTypes.element,
+  icon: PropTypes.any,
 };
```

Comment:
why are we changing it back to any?
I would have put icon: PropTypes.component

Reply:
Well we're technically passing a function as an icon for the trace folds.

![screen shot 2018-01-08 at 1 01 41 pm](https://user-images.githubusercontent.com/11803153/34687370-1fa6db3e-f474-11e7-85d1-7c911d49384a.png)

I haven't used `PropTypes.component` before, would that account for the function, too?

Reply:
If the function is a class constructor for a react component then `Proptypes.node` should work. (I think that is what @VeraZab is referring to there is no `.component` AFAIK.)

Reply:
Yeah I tried that -- still get `Failed prop type: Invalid prop `icon` supplied to `Fold`, expected a ReactNode.`

How about `icon: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),`?

Reply:
man, ok whatever works